usr/libexec/playtron/hardware-test-tool

```diff
@@ -391,6 +392,87 @@ class DisplayTest(PlaceholderTest):
             self.done = True
 
 
+class SpeakersTest(PlaceholderTest):
+    def __init__(self):
+        self.name = 'Speakers|扬声器'
+        self.reset()
+        self.start_time = None
+        self.time_count = 0
+        self.wav_file_name = '/tmp/test_record.wav'
+        self.msg_headphones_hint = 'Unplug headphones|拔下耳机'
+        self.audio_output_started = False
+        self.audio_input_started = False
+        self.cleanup_started = False
+
+    def start(self, screen, font):
+        self.reset()
+        self.start_time = time.perf_counter()
```

Comment:
should reset `audio_output_started`, `audio_input_started`, and `cleanup_started` here.
Without this you cannot run the test again.